.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+image: gitpod/workspace-ruby-3.1
+tasks:
+  - init: |
+      # Code fetch for generating command guide
+      git clone --depth 1 -b 3.3 https://github.com/rubygems/rubygems.git ../rubygems || true &
+      bundle config set --local path vendor/bundle
```

Comment:
`--local` is the default now, no need to specify it.

Reply:
I did not know that. Even bundler 2.3.16 warns as follows against `bundle install --path x`:
```
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local path 'x'`, and stop using this flag
```

Reply:
Indeed, we haven't yet updated the recommendation to stay on the safe side and recommend something that works consistently across versions, but that has changed now :)